themes/src/main/resources/theme/keycloak.v2/login/register.ftl

```diff
@@ -73,22 +73,23 @@
             ].filter(p => p.policy.value !== -1);
 
             document.getElementById("password").addEventListener("change", (event) => {
-                const serverErrors = document.getElementById("input-error-password");
-                if (serverErrors) {
-                    serverErrors.remove();
-                }
 
                 const template = document.querySelector("#errorTemplate").content.cloneNode(true);
-
                 const errors = validatePassword(event.target.value, activePolicies);
+
+                if (errors.length === 0) {
```

Comment:
Why do we need this? At the end the children are always replaced.

Reply:
If we don't remove the empty div tree, the `pf-v5-c-form__helper-text` class of the first div add's some unnecessary top-margin to the form.

Maybe there's a cleaner way to code this, but I think we should still remove that margin.

Reply:
I see your point, then I would suggest we call `replaceChilderen` without any arguments, that should clear out the elements. Additionally, let's store the element so we don't call `getElementById()` multiple times.

```js
const errorContainer = document.getElementById("input-error-container-password");

if (errors.length === 0) {
  errorContainer.replaceChildren();
  return;
}
```

Reply:
Sounds good 👍